Jenkinsfile

```diff
@@ -13,8 +13,8 @@ String appImageName           = 'docker.soramitsu.co.jp/sora2/substrate'
 String secretScannerExclusion = '.*Cargo.toml\$|.*pr.sh\$'
 Boolean disableSecretScanner  = false
 int sudoCheckStatus           = 0
-String featureList            = 'private-net include-real-files reduced-pswap-reward-periods'
-Map pushTags                  = ['master': 'latest', 'develop': 'dev','substrate-4.0.0': 'sub4']
+String featureList            = 'private-net include-real-files reduced-pswap-reward-periods wip ready-to-test'
+Map pushTags                  = ['master': 'latest', 'develop': 'dev','trustless-evm-bridge': 'bridge']
```

Comment:
Do we need to remove `trustless-evm-bridge` ?

Reply:
Will remove in separate PR after merge

Reply:
Ok